doc/source/whatsnew/v1.0.0.rst

```diff
@@ -97,6 +97,7 @@ Datetimelike
 ^^^^^^^^^^^^
 - Bug in :meth:`Series.__setitem__` incorrectly casting ``np.timedelta64("NaT")`` to ``np.datetime64("NaT")`` when inserting into a :class:`Series` with datetime64 dtype (:issue:`27311`)
 - Bug in :meth:`Series.dt` property lookups when the underlying data is read-only (:issue:`27529`)
+- Bug in :meth:`HDFStore.__getitem__` incorrectly reading tz attribute created in Py2 (:issue:`26443`)
```

Comment:
I don't think magic methods are in the docs.

It also seems like we don't have a generic `HDFStore` reference, so just write this as double backticks rather than :meth:.

Reply:
Ok, will commit again.